prqlc/prqlc/Cargo.toml

```diff
@@ -123,6 +123,12 @@ name = "bench"
 # refer to the path relative to `prqlc`, so we choose this crate out of
 # default.
 
+[[package.metadata.release.pre-release-replacements]]
+exactly = 1
+file = "../packages/snap/snapcraft.yaml"
+search = "^version: (git|'[\\w.\\-]+')$"
```

Comment:
Yeah, we can do that.

In theory it should be possible to use `git` as  version, I think the reason the version is `0` is because the GitHub Action [`action/checkout`](https://github.com/actions/checkout) only fetches the last commit and does not fetch any tags. It has a `fetch-tags` option that can be set to `true` but the `fetch-depth` option has the default value `1` but if set to `0` it will fetch all history for all branches and tags but that may take time on larger repositories.

Reply:
(zero rush but if what I'm saying is confusing, let me know (sometimes people say I can be confusing)! I'm happy to do the final change to this if that would be helpful. TY)

Reply:
No, you're not confusing, its just me who was hesitating on which approach to go with.